.github/workflows/main.yml

```
@@ -110,3 +110,21 @@ jobs:
         with:
           name: collection
           path: theforeman-operations-*.tar.gz
+
+  molecule:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6]
```

does this need to be a matrix? given we only test one version?

if yes, mind using 3.7, strings and a more verbose list notation?
```suggestion
        python-version:
          - "3.7"
```

---

I meant to ask you about that. The size of the matrix seemed a lot to spin up containers with full blown installs and so I started with picking a single python version to do a live test. Is that a sound strategy?

---

I don't think the results will vary based on Python version (we don't have any python code), so if anything, I'd test Ansible versions.